nixos/tests/cloud-init.nix

```diff
@@ -51,29 +51,33 @@ in makeTest {
     networking.hostName = "";
   };
   testScript = ''
-    machine.wait_for_unit("cloud-init.service")
-    machine.succeed("cat /tmp/cloudinit-write-file | grep -q 'cloudinit'")
+    unnamed.wait_for_unit("cloud-init.service")
```

Comment:
why the `s/machine/unnamed/`?

Reply:
Because otherwise it wouldn't work! The test script will just crash on the first line. It's because the objects for test machines in the namespace are generated from system closure hostnames.

Reply:
I was used to these being called `machine`, and the tests working, so mostly confused myself here. Seems like this is now needs to be called `unnamed` here, where `machine` still works in other places, like `nixos/tests/sudo.nix` :shrug

Reply:
It's probably because the hostname is unset now, and now the closure, instead of having a hostname after its name in the attribute set, now has a hostname equal to an empty string. NixOS handles this by tagging it as "unnamed".